### DISCUSSION TITLE
Any advice to not get terminated?

### DISCUSSION THREAD

**Discussion by Zenixas** [Vedza/NitroSniperGo#211] [General]
Hey, I just won a giveaway and I was so excited, but 20 minutes later my account got terminated, so I basically lost it :/
 How do I avoid this from happening again?

**Comment by Vedza** [comment]
People probably reported you, so we can't to anything about that except if you put a big delay to join giveaway or disable giveaway joiner if you are in too many servers that does giveaways.

**Reply by Zenixas** [reply]
Yeah, thanks. Also do you think there could be higher chance of getting banned if I'm running this thing on a Vps 24/7?

**Comment by DarkGD123** [comment]
I avoid the termination by:

• Turning the Sniper on when i wake up and Turning it off when i go to sleep
• I limit my nitro sniping to 1 and it will cooldown to 24-48hrs
• Setting some useless stuff to `false` in `settings.json`
